packages/product-react/src/Tabs/Tabs.tsx

```diff
@@ -0,0 +1,217 @@
+import classnames from 'classnames';
+import * as React from 'react';
+import { Button } from '@wework/ray-core-react';
+import { TabContext } from './context';
+
+import './Tabs.scss';
+
+interface ITabRefs {
+  [key: string]: React.RefObject<HTMLButtonElement>;
+}
+
+export interface ITab {
+  id: string;
```

Comment:
I'm assuming that the ID will most likely be a string as well and somebody won't just use their index. 😂